components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -94,12 +94,39 @@
   #set ($collaborators = $request.getParameterValues('collaborator'))
   #set ($accessLevels = $request.getParameterValues('accessLevel'))
   #set ($owner = "$!{request.get('owner')}")
+  #if ("$!{owner}" != "")
+    ##check the validity of the new owner
+    #set ($invalidOwner = false)
+    #set ($validGroup = false)
+    #set ($validUser = false)
+    #set ($cannotTransfer = false)
+    #set ($ownerDocRef = $services.model.resolveDocument("$owner"))
+    #if ($xwiki.getDocument($ownerDocRef).getObject("PhenoTips.PhenoTipsGroupClass"))
```

Comment:
For performance reasons, if you don't have any variables to interpolate in a string, in Velocity code it is better to use single quotes.